recipes/05/05_group_by_window.md

```diff
@@ -0,0 +1,50 @@
+# 05 Aggregating Time Series Data
+
+:bulb: This example will show how to aggregate time-series data in real-time using a `TUBMLE` window.
+
+The source table (`server_logs`) is backed by the [`faker` connector](https://github.com/knaufk/flink-faker), which continuously generates rows in memory based on Java Faker expressions.
+
+Many streaming applications work with time-series data.
+To count the number of `DISTINCT` ip-addresses seen each minute, rows need to be grouped based on a [time attribute](https://docs.ververica.com/user_guide/sql_development/table_view.html#time-attributes).
+Grouping based on time is special, because time always moves forward, which means Flink can generate final results after the minute is completed. 
+
+`TUMBLE` is a built-in function for grouping timestamps into intervals.
```

Comment:
```suggestion
`TUMBLE` is a built-in function for grouping timestamps into time intervals called windows.
```

Reply:
I think, we need to introduce this terminology otherwise we will be screwed later on when we rely on it to be concise.